glustercli/cmd/cli_utils_test.go

```diff
@@ -0,0 +1,16 @@
+package cmd
```

Comment:
Please name this file as `utils_test.go`.

By convention, for a source file named `X.go`, unit tests are placed in `X_test.go`

Reply:
done